src/en/charms-deploying.md

```diff
@@ -281,6 +281,46 @@ are supplied, then the remaining units will be assigned as normal to a new,
 clean machine.
 
 
+## Deploying with binding
+
+You can configure more complex networks using [spaces](./network-spaces.html). This
+also permits the use of `--bind` to specify which network space individual
+charm endpoints should use. When you bind, the endpoint(s) of the charm will
+have IP addresses from subnets that are part of the specified space.
+
+When `--bind` is not used, all endpoints will use the host machine's preferred
+private address, which you can see using `unit-get private-address`.
+
+Create your space before deploying a charm that attempts to use it.
```

Comment:
I'm going to move the sentence up to the opening paragraph. It still fits there, but makes the link between the two topics more obvious.